terraform/environments/xhibit-portal/network-infrastructure.tf

```diff
@@ -226,6 +232,27 @@ resource "aws_security_group_rule" "waf_lb-outbound-importmachine" {
   source_security_group_id = aws_security_group.importmachine.id
 }
 
+resource "aws_security_group_rule" "prtg_lb-inbound-importmachine" {
+  depends_on               = [aws_security_group.prtg_lb]
+  security_group_id        = aws_security_group.prtg_lb.id
+  type                     = "ingress"
+  description              = "allow all from bastion"
```

Comment:
Can you correct these descriptions? This is allowing access from the importmachine to the loadbalancer

Reply:
description updated= "allow all from prtg-lb to importmachine"